plugins/modules/clustering/nomad/nomad_job.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
```

Comment:
```suggestion
author: FERREIRA Christophe (@chris93111)
```
according to the Ansible doc